components/private/XCTest/src/XCTest+MaterialColor.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+@interface XCTestCase (MaterialColor)
+
+- (void)assetEqualFirstColor:(UIColor *)firstColor secondColor:(UIColor *)secondColor;
```

Comment:
typo: asset -> assert

How about naming it `assertEqualColors` similar to https://developer.apple.com/documentation/xctest/xctassertequalobjects naming

Reply:
Good idea. I added a macro to match the names (`XCTAssertEqualColors`). Addressed in fbd7964f7bab204a254d00d8e2690c503cf53ebd.